pkg/detector/compare_test.go

```diff
@@ -78,12 +81,20 @@ func Test_GetHigherPriorityPropagationPolicy(t *testing.T) {
 				},
 			},
 		},
+		{
+			name: "Test 4",
```

Comment:
We'd better give each case a human-readable name so that we can go through and check the coverage easier.

Reply:
Updated.